swarm/swarm.go

```diff
@@ -262,8 +277,7 @@ func (s *Swarm) control() {
 		case req := <-s.getOutgoing:
 			s.messages = takeMaxLatest(s.messages, req.overhead, req.limit)
 			if len(s.messages) <= 0 {
-				time.Sleep(10 * time.Millisecond)
-				continue
+				log.Warning("SWARM: getOutgoing with 0 messages, should not happen")
```

Comment:
this warning text may be misleading because the reason for 0 messages can be simply that the underlying messaging cannot send out the next message if it is too large. I think a better warning text would be: "0 outgoing messages with the current overhead and queued message size"